cmd/checkmarxExecuteScan_generated.go

```diff
@@ -216,7 +218,6 @@ func addCheckmarxExecuteScanFlags(cmd *cobra.Command) {
 	cmd.Flags().IntVar(&myCheckmarxExecuteScanOptions.VulnerabilityThresholdMedium, "vulnerabilityThresholdMedium", 100, "The specific threshold for medium severity findings")
 	cmd.Flags().StringVar(&myCheckmarxExecuteScanOptions.VulnerabilityThresholdResult, "vulnerabilityThresholdResult", "FAILURE", "The result of the build in case thresholds are enabled and exceeded")
 	cmd.Flags().StringVar(&myCheckmarxExecuteScanOptions.VulnerabilityThresholdUnit, "vulnerabilityThresholdUnit", "percentage", "The unit for the threshold to apply.")
-	cmd.Flags().BoolVar(&myCheckmarxExecuteScanOptions.Verbose, "verbose", false, "Whether the step shall provide verbose logging output")
```

Comment:
@nevskrem looks like this went into master by accident. Maybe we should add `go run pkg/generator/step-generator.go` as a pre-commit hook.

Reply:
Having such pre-commit hooks updating the code "on the fly" during the commit means we cannot control what we are checking in. That's actually bad from my point of view.

Would prefer better reviews.

Other approach: what about re-generating and checking is something has been changed by that re-generate process alongside with aborting the commit in this case.

Reply:
I would strive for including this into the test process somehow.